<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Serial</class>
 <widget class="QMainWindow" name="Serial">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>646</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>串口调试助手kpp</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QGroupBox" name="RecGroupBox">
      <property name="title">
       <string>接收数据</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QTextBrowser" name="textBrowser"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1" rowspan="2">
     <widget class="QGroupBox" name="PortGroupBox">
      <property name="title">
       <string>串口操作</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="5" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3"/>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>串口号：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>波特率：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QComboBox" name="portNameComboBox">
         <item>
          <property name="text">
           <string>COM1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM6</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM7</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM8</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM9</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM10</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QComboBox" name="stopBitsComboBox">
         <item>
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1.5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QComboBox" name="baundRateComboBox">
         <item>
          <property name="text">
           <string>9600</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4800</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>14400</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>19200</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>38400</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>43000</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>57600</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>115200</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>128000</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>230400</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>校验位：</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>停止位：</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>数据位：</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QComboBox" name="dataBitsComboBox">
         <item>
          <property name="text">
           <string>8</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>7</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>6</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>5</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QComboBox" name="pariatyComboBox">
         <item>
          <property name="text">
           <string>无</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>奇</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>偶</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="6" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_5"/>
       </item>
       <item row="6" column="2">
        <widget class="QPushButton" name="closePoetBtn">
         <property name="text">
          <string>关闭串口</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QPushButton" name="OpenPortBtn">
         <property name="text">
          <string>打开串口</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0" rowspan="2">
     <widget class="QGroupBox" name="verticalGroupBox_2">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QCheckBox" name="HexcheckBox">
         <property name="text">
          <string>十六进制发送</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="ReccheckBox">
         <property name="text">
          <string>十六进制显示</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="obocheckBox">
         <property name="text">
          <string>定时发送</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="delayspinBox">
         <property name="minimum">
          <number>10</number>
         </property>
         <property name="maximum">
          <number>5000</number>
         </property>
         <property name="value">
          <number>200</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;ms&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>保存窗口</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="1" rowspan="2">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>数据操作</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QPushButton" name="clearRecBtn">
         <property name="text">
          <string>清空接收</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="clearSendBtn">
         <property name="text">
          <string>清空发送</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QGroupBox" name="sendGroupBox">
      <property name="title">
       <string>发送数据</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="sendMsgBtn">
         <property name="text">
          <string>单行发送</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>646</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="contrlMenu">
    <property name="title">
     <string>操作</string>
    </property>
   </widget>
   <widget class="QMenu" name="aboutMenu">
    <property name="title">
     <string>关于</string>
    </property>
   </widget>
   <addaction name="contrlMenu"/>
   <addaction name="aboutMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
